client/src/graphics/base.rs

```diff
@@ -139,7 +139,11 @@ impl Base {
                             .queue_create_infos(&[vk::DeviceQueueCreateInfo::default()
                                 .queue_family_index(queue_family_index)
                                 .queue_priorities(&[1.0])])
-                            .enabled_extension_names(&device_exts),
+                            .enabled_extension_names(&device_exts)
+                            .push_next(
+                                &mut vk::PhysicalDeviceDescriptorIndexingFeatures::default()
```

Comment:
I chose this because it best matched the documentation:
> This crate requires at least Vulkan 1.2 and a GPU with support for `VkPhysicalDeviceDescriptorIndexingFeatures.descriptorBindingPartiallyBound`

Strangely, based on https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkPhysicalDeviceDescriptorIndexingFeatures.html and https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkPhysicalDeviceVulkan12Features.html, both seem to be provided by Vulkan 1.2, suggesting they were introduced at the same time, so I'm not sure why this redundancy exists.